pkg/certificate/mrc_reconciler.go

```diff
@@ -57,16 +60,19 @@ func (m *Manager) handleMRCEvent(event MRCEvent) error {
 	log.Err(err).Msg("error setting issuers on mrc")
 	// TODO(5046): improve logging in this method.
 	if shouldUpdateMRCComponentStatus(mrc) {
```

Comment:
When the initial certificate is created, it will flow through this event three times?

Reply:
Do you mean the initial MRC on start up?

Reply:
something like a new mrc: `Kubectl apply -f new-mrc.yaml`

Reply:
If I am reading this correctly, a user creates a new MRC with `intent=passive` then then the componentstatus is set, this triggers a change firing another event, which then sets the state which then fires another event which finally sets the issuers?  Otherwise this if statement won't do anything other than set componentstatus on the first submision by the user since all these if's are nested.

It seems to me we should be able to do all that in go?